src/components/twilio-voice-dialer/routes.js

```diff
@@ -0,0 +1,64 @@
+import { Router } from 'express';
+import crypto from 'crypto';
+import Twilio from 'twilio';
+import config from '../../config.js';
+
+const AccessToken = Twilio.jwt.AccessToken;
+const VoiceGrant = AccessToken.VoiceGrant;
+const VoiceResponse = Twilio.twiml.VoiceResponse;
+
+const router = Router();
+const { appSid, accountSid, apiKeySid, apiKeySecret, callerId, callbackBaseURL } = config;
+const client = Twilio(apiKeySid, apiKeySecret, { accountSid });
+
+router.get('/token', (req, res) => {
+  const identity = req.query.identity;
+  const token = new AccessToken(accountSid, apiKeySid, apiKeySecret, { identity, ttl: 3600 });
+  const voiceGrant = new VoiceGrant({
+    outgoingApplicationSid: appSid,
+    incomingAllow: !!identity,
```

Comment:
Since we're starting fresh, I would prefer if we can use `Boolean(identity)` and consistently keep that through the codebase. I know we don't do this in the Voice JS SDK for consistency's sake.